src/isar_exr/api/energy_robotics_api.py

```diff
@@ -317,164 +289,206 @@ def wake_up_robot(
                     f"Not able to wake up robot after '{timeout}' seconds."
                 )
             sleep(1)
+        print("success waking up robot")
```

Comment:
This should be part of a logger rather than a print

Reply:
Was from local testing, had forgotten to remove it